modules/sdk-api/src/types.ts

```diff
@@ -275,3 +276,18 @@ export interface RegisterPushTokenOptions {
   pushToken: unknown;
   operatingSystem: unknown;
 }
+
+export interface LoginResponse {
+  // The API session route does not return this. It's annotated by the SDK
+  access_token?: string;
+  derivationPath: string;
+  encryptedECDHXprv?: string;
+  encryptedToken?: string;
+  // Unit timestamp of expiration
+  expires_at: number;
+  // seconds in which the token will expire from issuance
+  expires_in: number;
+  scope: string[];
+  token_type: string;
+  user: unknown;
+}
```

Comment:
Nit: It would be nice to have this as two types/interfaces. One for what the API returns and an extension of that for what the SDK annotates.

Reply:
i dont think we add anything extra (not super familiar with all the flow), the user settings we just update the field with the new user settings